config/jobs/kubernetes/sig-k8s-infra/trusted/sig-security-trusted.yaml

```diff
@@ -91,3 +91,28 @@ periodics:
     testgrid-num-failures-to-alert: '1'
     testgrid-dashboards: sig-security-snyk-scan
     description: Run snyk scan on k/k master periodically
+- name: auto-refreshing-official-cve-feed
+  interval: 2h
+  cluster: k8s-infra-prow-build-trusted
+  decorate: true
+  extra_refs:
+  - org: kubernetes
+    repo: test-infra
```

Comment:
@nehaLohia27 if we switch this to `sig-security` we can move the hack scripts into `k/sig-security/sig-security-tooling/cve-feed/hack`. That should resolve the suggestion from @ameukam from here: https://github.com/kubernetes/test-infra/pull/26896#issuecomment-1194737954

(Inspired by @palnabarun doing something similar here: https://github.com/kubernetes/test-infra/pull/26928/files#diff-3d28e7d52d993a47561abb62878a9b198ff9a80ebec4afd014a7e754f9e39363R2)

Reply:
I was going to suggest the same here. But, thought there might be reasons behind keeping the script here. 😊

Reply:
Updated